.copr/prep.sh

```diff
@@ -11,4 +11,6 @@ sed "s,#COMMIT#,${COMMIT},;
      s,#COMMITDATE#,${COMMIT_DATE}," \
          packaging/fedora/ksh.spec.in > packaging/fedora/ksh.spec
 
+sed -i "s/#define SH_RELEASE.*/#define SH_RELEASE \"${COMMIT_DATE}+git.${COMMIT_NUM}.${COMMIT_SHORT}\"/" src/cmd/ksh93/include/version.h 
```

Comment:
That seems needlessly verbose to me but it otherwise looks fine. And since this will probably be changed in response to issue #335 it's find for now.